application/classes/Ushahidi/Repository/Form/Stats.php

```diff
@@ -87,6 +88,114 @@ public function getResponses($form_id)
 			->execute($this->db)
 			->get('total');
 	}
+
+	/**
+	 * @param $form_id
+	 * @param $total_sent
+	 * @return float|int
+	 * Returns the number of TOTAL pending questions (for invalidated+valid contacts off a
+	 * targeted_survey)
+	 */
+	public function countTotalPending($form_id, $total_sent) {
```

Comment:
I think these queries are correct, I'm glad that they are in sql because it makes it slightly more clear, I'm sad we don't have ORM to make these kinds of things slightly easier. I ran the queries only in my head.

Do you have a data set to run them against?